movies/0.index.md

```diff
@@ -7,7 +7,7 @@ The Hydra Movies API shows how a basic CRUD (Create, Read, Update, Delete) API c
 
 The following recipes decribe Hydra features that are used by the API.
 
-  * [1 API entry point](1.entry-point.md)
-  * [2 The JSON-LD context](movies/2.context.md)
-  * [3 The API documentation](movies/3.api-doc.md)
-  * [4 Create a movie](movies/4.create-movie.md)
+  * [1 API entry point](./1.entry-point.md)
```

Comment:
I'm not sure I understand - are we dropping numbers only in specific places or everywhere?

Reply:
I think I didn't want to change stuff unrelated to the "Api Documentation" section.

If we agree here, next PRs would cover the movies example and possible the old drafts